schema/motor-insurance.json

```diff
@@ -27,7 +27,26 @@
             "description": "Client's consent for final price, should exactly match the <code>finalPrice</code> object from output.<br/>Automation will not proceed with placing order until the consent is provided."
         },
         "vehicle": {
-            "typeRef": "#/domains/MotorInsurance/types/Vehicle"
+            "typeRef": "#/domains/MotorInsurance/types/Vehicle",
+            "dataGenerationInstruction": {
+                "method": "DataGenerationApi",
+                "generatorRequestSchema": {
+                    "type": "object",
+                    "properties": {
+                        "vehicle": {
```

Comment:
@qubyte this project is built based on this spec: https://github.com/universalbasket/data-generation/blob/master/SPEC.md#post-domainsmotorinsurancetypesvehicle

I think of this as a boundary between involved parties, so it doesn't really belongs to neither of data generation service nor its client (job input bundler). Both client and server need to know about this schema, server in order to validate it before saving, client in order to validate it before sending to service api.